### ISSUE TITLE
Move the zoom bar from top of chart to bottom of chart

### ISSUE THREAD

**Issue by tifftea** [carbon-design-system/carbon#5876]
Move the zoom bar from top of chart to bottom of chart

### Summary

The zoom bar should be moved below the chart to make room for important subtext, metrics, and the chart legend.

Before: 

![zoom bar before](https://user-images.githubusercontent.com/63752979/79475404-7911a480-7fbc-11ea-822f-8e7be3f03141.png)

After:

![chart after](https://user-images.githubusercontent.com/63752979/79475374-71520000-7fbc-11ea-9a01-6465c82e0c8f.png)

### Justification

Users' mental processing was oriented on the information hierarchy and they tried to comprehend the information metric system from top to bottom. 
Below is an example of the order of the mental processing: 
1. What is it (title)?
2. What does it mean (title definition)?
3. What are x and y axises represent? 
4. What does each line mean? 
5. What is the trend?
6. What happened at this point?
7. Let me zoom into it to investigate further.

The current CX lacks component to support DB2 WoC cases. 

Users’ cognition process on data chat is as follows:
The cursor mirrors users’ attention focal point in the data chart and pinpoints the location of the integrating the X and Y axis information in cognitive process. Once oriented in the data environment, users inspect more granular information (to go deeper and reaches cognition decision in the cognition tree) by correlating X and Y axis over time progression and regression.
Based on the human factor design guideline, under the graphic modules is identified as the optimal location in order to prevents potential minor or cosmetic usability errors to meet users’ cognition progression as follows:
Simple zoom interaction, precise zoom interaction, pinpointing/impeccable zoom interaction

### Desired UX and success metrics

--

### "Must have" functionality

--

### Specific timeline issues / requests

--

### Available extra resources

--

**Comment by emyarod** [comment]
This issue may be more relevant over in https://github.com/carbon-design-system/carbon-charts

unfortunately it looks like I'm not able to transfer this issue over to that repository
